src/main/java/com/Forecast/Forecast/weather/CorsConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.Forecast.Forecast.weather;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class CorsConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
```

Comment:
![image](https://github.com/Radgor144/forecast/assets/47002105/8faaf7ef-ee89-4edd-88f0-0d9f767966b2)
to wszystko powinno być zaczytywane z propertek z ymla